packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -203,17 +204,21 @@ async function instantiateModule(
     )
   } catch (e) {
     mod.ssrError = e
+
     if (e.stack && fixStacktrace) {
       ssrFixStacktrace(e, moduleGraph)
-      server.config.logger.error(
-        `Error when evaluating SSR module ${url}:\n${e.stack}`,
-        {
-          timestamp: true,
-          clear: server.config.clearScreen,
-          error: e,
-        },
-      )
     }
+
+    server.config.logger.error(
+      `Error when evaluating SSR module ${url}:` +
+        (e.importee ? ` failed to import '${e.importee}'\n` : '\n'),
```

Comment:
Also, it might be a nice opportunity here to mark this with red color so it stands out between the nodejs error 😬